package.json

```diff
@@ -73,6 +74,7 @@
     "jest-localstorage-mock": "^2.4.12",
     "lint-staged": "^11.0.0",
     "nanodelay": "^2.0.0",
+    "preact": "^10.5.13",
```

Comment:
Do we need to add optional peer dependency as we did for `react`?

Reply:
I think yes, but not sure which version it should be